spec/rails_helper.rb

```diff
@@ -0,0 +1,65 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
```

Comment:
- [ ] Please add more tests for both unit and integration which cover more in-depth your app